presentation/src/main/java/com/stop/ui/util/Marker.kt

```diff
@@ -0,0 +1,27 @@
+package com.stop.ui.util
+
+import android.graphics.Color
+import com.stop.R
+
+object Marker {
+    const val PLACE_MARKER = "place_marker"
+    const val PLACE_MARKER_IMG = R.drawable.ic_point_marker
+
+    const val PERSON_MARKER = "marker_person_pin"
+    const val PERSON_MARKER_IMG = R.drawable.ic_person_marker
+    const val PERSON_LINE = "person_line"
+    const val PERSON_LINE_COLOR = Color.MAGENTA
+
+    const val BOOKMARK_MARKER_IMG = R.drawable.ic_bookmark_marker
+
+    const val BUS_MARKER = "marker_bus_pin"
+    const val BUS_MARKER_IMG = R.drawable.ic_bus_marker
+    const val BUS_LINE = "bus_line"
+    const val BUS_LINE_COLOR = Color.BLUE
+
+    const val SUBWAY_MARKER = "marker_subway_pin"
```

Comment:
```suggestion
    const val SUBWAY_MARKER = "subway_marker"
```
제가 다시 바꿔놨던 건데 이런 식으로 다 통일하면 좋을 것 같습니다.

Reply:
https://github.com/boostcampwm-2022/android10-PlzStop/pull/58/commits/b5d0d22627017efded3ed16594b29d32f90f2944